jsx/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "jupyterhub-admin-react",
+  "version": "1.0.0",
+  "description": "React application for the Jupyter Hub admin dashboard service",
+  "main": "index.js",
+  "author": "nabarber",
+  "license": "MIT",
```

Comment:
I get this error:

```
warning ../package.json: License should be a valid SPDX license expression
```.

Most code in the JupyterHub ecosystem is BSD-3. Would you consider using that instead of MIT?

Reply:
Fixing in next commit 👍

Reply:
Setting them to `BSD-3-Clause` seems to fix the warning